pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py

```diff
@@ -1037,9 +1056,12 @@ def process_setting(setting):
         production_render_setting_categories = [category for category in render_setting_categories if category['name'] != 'ViewportSettings']
         generate_houdini_ds(install_path, 'Global', production_render_setting_categories)        
         viewport_render_setting_categories = [category for category in render_setting_categories \
-            if category['name'] == 'Sampling' or category['name'] == 'AdaptiveSampling' or category['name'] == 'Denoise' or category['name'] == 'ViewportSettings']
+            if category['name'] == 'RenderQuality' or category['name'] == 'Sampling' or category['name'] == 'AdaptiveSampling' or category['name'] == 'Denoise' or category['name'] == 'ViewportSettings']
         for category in viewport_render_setting_categories:
-            del category['houdini']
+            if category['name'] == 'RenderQuality':
+                for setting in category['settings']:
+                    if setting['name'] == 'core:renderQuality':
```

Comment:
can it be simplified like this?
`viewport_render_setting_categories = (category for category in render_setting_categories \`
`        if category['name'] in ('RenderQuality', 'Sampling', 'AdaptiveSampling', 'Denoise', 'ViewportSettings'))`
`for category in (cat for cat in viewport_render_setting_categories if cat['name'] == 'RenderQuality'):`
`    for setting in (set for set in category['settings'] if set['name'] == 'core:renderQuality'):`
`        ...`